README.md

```diff
@@ -810,9 +810,14 @@ The `ghw.PCIDevice` struct has the following fields:
 #### Finding a PCI device by PCI address
 
 In addition to the above information, the `ghw.PCIInfo` struct has the
-following method:
+following methods:
 
 * `ghw.PCIInfo.GetDevice(address string)`
+* `ghw.PCIInfo.LookupDevice(address string)`
+
+The difference between the two methods is that `LookupDevice` will use
+cached data, while `GetDevice` will attempt every time to get informations
+from the system.
```

Comment:
The caching behaviour of any `ghw` struct should be entirely transparent to the user, so I'm going to push back on this particular change. The user should not need to know the internal implementation details in order to know to call a different method of `ghw.PCIInfo` depending on whether an underlying sysfs/filesystem mount is present.

In other words, just change the `PCIInfo.GetDevice` method implementation to look up the device in `PCIInfo.Devices` by address before searching a filesystem :)

Reply:
very good point.